spring-data-jdbc/src/test/java/org/springframework/data/jdbc/repository/JdbcRepositoryIntegrationTests.java

```diff
@@ -923,6 +913,19 @@ void findAllByExamplePageable(Pageable pageRequest, int size, int totalPages, Li
 		}
 	}
 
+	@Test
+	void findByExampleWithCollection() {
```

Comment:
Wouldn't it be simpler to test `RelationalExampleMapper` directly instead of adding another integration test that takes quite some time? We do have also QbE support for R2DBC and with this test we could argue that the R2DBC counterpart is missing.

Reply:
Done